.circleci/bin/test-airflow-image.py

```diff
@@ -251,6 +251,12 @@ def test_airflow_configs(scheduler, docker_client):
                 "cat /usr/lib/python3.7/site-packages/airflow/config_templates/default_airflow.cfg | "
                 "grep '^run_as_user' | awk '{print $3}'").strip() == ""
 
+    if semantic_version(airflow_version) >= semantic_version('1.10.10'):
```

Comment:
This flag is only available from 1.10.7

https://airflow.readthedocs.io/en/latest/configurations-ref.html#update-fab-perms